README.md

```diff
@@ -121,6 +122,28 @@ key: "DISABLE_OPTIMIZED_EXECUTION"
 }
 ```
 
+* `INFERENCE_MODE`: Boolean flag to enable the Inference Mode execution
+of TorchScript models. By default the inference mode is disabled. 
+
+[InferenceMode](https://pytorch.org/cppdocs/notes/inference_mode.html) is a new
+RAII guard analogous to NoGradMode to be used when you are certain your operations
+will have no interactions with autograd. Compared to NoGradMode, code run under
+this mode gets better performance by disabling autograd.
+
+Please note that in some models, InferenceMode might not benefit performance
+and in fewer cases might impact performance negatively.
+
+The section of model config file specifying this parameters will look like:
+
+```
+parameters: {
+key: "INFERENCE_MODE"
```

Comment:
If inference mode is going to be the official way for PyTorch inferencing, shouldn't it be set by default unless user want the model to be run in NoGradMode?

Reply:
I was going to make this the default to keep to the behavior in the python and C++ executions users must enable the same. 

In the future if we see that the InferenceMode consistently improves performance we can enable it by default. For now I would prefer it be an opt in flag instead of opt out.

Reply:
I agree it should be off by default. In general we should prefer to maintain backwards compat even in perf until we have high confidence that the new mode is better.